packages/components/organisms/f-footer/README.md

```diff
@@ -65,31 +72,63 @@ Global Footer Component for Vue.js.
 
     This component extends [@justeat/browserslist-config-fozzie](https://github.com/justeat/browserslist-config-fozzie).
 
+## Configuration
+
+### Props
+
+There may be props that allow you to customise its functionality.
+
+The props that can be defined are as follows (if any):
+
+| Prop  | Type  | Default | Description |
+| ----- | ----- | ------- | ----------- |
+
+### CSS Classes
+
+The modal has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.
+
+In addition to this, the modal exposes some classes which you can target in your application.
```

Comment:
```suggestion
The footer has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.

In addition to this, the footer exposes some classes which you can target in your application.
```